.github/ISSUE_TEMPLATE/enhancement_proposal.yaml

```diff
@@ -0,0 +1,33 @@
+name: Enhancement Proposal
+description: File an enhancement proposal
+labels: ["enhancement"]
+body:
+  - type: markdown
```

Comment:
should we also create an option to select which of the 4 core charms the enhancement proposal is for? additionally, this can also be used to add a label to the enhancement proposal issue